edge/pkg/edged/edged.go

```diff
@@ -957,6 +957,11 @@ func (e *edged) consumePodAddition(namespacedName *types.NamespacedName) error {
 
 	e.workQueue.Enqueue(pod.UID, utilwait.Jitter(time.Minute, workerResyncIntervalJitterFactor))
 	klog.Infof("consume added pod [%s] successfully\n", podName)
+	// update pod status after consume added pod, so the pod's metric data could be acquired
+	if err := e.updatePodStatus(pod); err != nil {
+		klog.Errorf("consumePodAddition: update pod %s status error", pod.Name)
+		return err
+	}
```

Comment:
Thanks for the fixing! @lvchenggang I just tested in my env, really a bug. Now we only update podstatus from PLEG events. For pod from cloud, pleg can catch the runtime event and update pod status. 

But when edgecore restarted, edged will get all pods info from metamanager, Ref: [edge.go#L1032](https://github.com/kubeedge/kubeedge/blob/master/edge/pkg/edged/edged.go#L1032), and have no pleg events now, so I think we can update the podstatus at here.

Reply:
It is really better to update podstatus in handlePodListFromMetaManager function, and I tried in my env prove it can work